src/index.js

```diff
@@ -20,10 +20,123 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0];
+
+const operators = {
+  add: '+',
+  subtract: '-',
+  multiply: '*',
+  divide: '/',
+};
+
+const operatorsMethods = {
+  add(beforeNumber, nextNumber) {
+    return beforeNumber + nextNumber;
+  },
+  subtract(beforeNumber, nextNumber) {
+    return beforeNumber - nextNumber;
+  },
+  multiply(beforeNumber, nextNumber) {
+    return beforeNumber * nextNumber;
+  },
+  divide(beforeNumber, nextNumber) {
+    return beforeNumber / nextNumber;
+  },
+};
+
+function render(
+  {
+    beforeNumber, nextNumber, operator, result,
+  } = {
```

Comment:
```suggestion
function render({
    beforeNumber, nextNumber, operator, result,
  }) {
```

매개 변수가 많은 경우 코드 정리를 이렇게 해주시는게 더 좋아요.